src/commands/sqlite.rs

```diff
@@ -437,3 +448,83 @@ pub fn find_database_link(db: &Database, label: &str) -> Option<Link> {
 pub fn database_has_link(db: &Database, link: &ResourceLabel) -> bool {
     db.links.iter().any(|l| l == link)
 }
+
+#[cfg(test)]
+mod sqlite_tests {
+    use super::*;
+    use cloud::MockCloudClientInterface;
+
+    #[tokio::test]
+    async fn test_create_error_already_exists() -> Result<()> {
+        let command = CreateCommand {
+            name: "db1".to_string(),
+            common: Default::default(),
+        };
+        let dbs = vec![
+            Database::new("db1".to_string(), vec![]),
+            Database::new("db2".to_string(), vec![]),
+        ];
+
+        let mut mock = MockCloudClientInterface::new();
+        mock.expect_get_databases().return_once(move |_| Ok(dbs));
+
+        let result = command.run(mock).await;
+        assert_eq!(
+            result.unwrap_err().to_string(),
+            r#"Database "db1" already exists"#
+        );
+        Ok(())
+    }
+
+    #[tokio::test]
+    async fn test_create_success() -> Result<()> {
```

Comment:
```suggestion
    async fn test_create_if_db_does_not_exist_db_is_created() -> Result<()> {
```

(to signal that the important thing here is not so much that the command returns Ok(()) but that `create_database()` gets called)